java/src/main/java/com/scalar/admin/AdminCommand.java

```diff
@@ -124,4 +156,34 @@ static String getCurrentTimeWithFormat() {
     Instant instant = Instant.now();
     return instant + " UTC (" + instant.toEpochMilli() + ")";
   }
+
+  private RequestCoordinator createCoordinator(String srvServiceUrl) {
+    return tlsEnabled
+        ? new TlsRequestCoordinator(srvServiceUrl, getCaRootCert(), overrideAuthority)
+        : new RequestCoordinator(srvServiceUrl);
+  }
+
+  private RequestCoordinator createCoordinator(List<InetSocketAddress> addresses) {
+    return tlsEnabled
+        ? new TlsRequestCoordinator(addresses, getCaRootCert(), overrideAuthority)
+        : new RequestCoordinator(addresses);
+  }
+
+  private String getCaRootCert() {
+    String caRootCert = null;
+
+    if (caRootCertPem != null) {
+      caRootCert = caRootCertPem.replace("\\n", System.lineSeparator());
```

Comment:
in case the user pass PEM formatted file in one line

Reply:
@kota2and3kan 
FYI. The error we had yesterday is because picocli take `\n` as a plaintext but not a new line character. So we need to convert it ourselves.

Reply:
@supl 
Ah, I see. I understood.
Thank you for your investigation!

Reply:
@supl 
Is it really picocli thing?
I'm not sure, but it might be shell thing.

I haven't run this command, but did a quick check with zsh and bash as follows. (Do you use bash?)
```
# zsh
$ echo "aaa\nbbb"
aaa
bbb

# bash
bash-3.2$ echo "aaa\nbbb"
aaa\nbbb
```

Reply:
@feeblefakie Just to be precise, the example you provided is caused not by the difference of the shells but by the difference in the `echo` commands. Both in bash and zsh, `echo` is a built-in command (you can see `type echo`). And, only the `echo` of zsh interpreters bacslashed escape characters, like "\n". We need the `-e` flag for bash's echo. See man of [bash](https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-echo) and [zsh](https://zsh.sourceforge.io/Doc/Release/Shell-Builtin-Commands.html).

So, I suppose when passing string parameters to this command, the difference between  bash and zsh does not affect.

Reply:
You can see that the strings passed to the programs are the same:

```
# zsh
❯ ruby -e "puts ARGV" "foo\nbar"
foo\nbar

# bash
❯ bash
bash-5.2$ ruby -e "puts ARGV" "foo\nbar"
foo\nbar
```

Reply:
@supl Sorry that I found that you already replied after reloading

Reply:
@choplin 
No problem at all.
Thank you very much for clarifying this question 🙇

Reply:
@supl @choplin Thank you for the clarification!